### PR TITLE
Create LIT6946InvocationMary (as in BLorient4930)

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a LIT for LIT6946InvocationMary according to our discussion in https://github.com/BetaMasaheft/Documentation/issues/2501.

### CODE REVIEW

**Review by eu-genia — APPROVED** [review]

File: new/LIT6946InvocationMary.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6946InvocationMary" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Invocation to the Virgin Mary and Jesus Christ for protection</title>
+                <title xml:lang="gez" xml:id="t2">ሰአሊ፡ ለነ፡ ቅድስት፡ ድንግል፡ ማርያም።</title>
+                <title xml:lang="gez" type="normalized" corresp="#t2">saʾali lana qǝddǝst dǝngǝl Māryām</title>
+                <title xml:lang="en" type="normalized" corresp="#t2">Pray for us, Holy Virgin Mary!</title>
```
Comment (thea-m) [inline new/LIT6946InvocationMary.xml:11]: To me, the exclamation mark seems unusual :)

**Review by thea-m — APPROVED** [review]

File: new/LIT6946InvocationMary.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT6946InvocationMary.xml:11]: I agree and I will delete it.

**Comment by thea-m** [comment]
Has this been closed without merging?

**Comment by CarstenHoffmannMarburg** [comment]
I reopened, merged and closed again. I hope, everything is fine, now?

**Comment by thea-m** [comment]
I think it is, I can see the record on the master branch

**Pull request merged**